frontend/src/components/Shared/Nav.js

```diff
@@ -36,9 +39,32 @@ class Nav extends Component {
             this.props.logout();
             this.setState({logout:true});
         }
-       
+    
+    openSignupForm=()=>{
+        this.setState({signupFormIsOpen: true});
+        }
+    
+    closeSignupForm=()=>{
+        this.setState({signupFormIsOpen: false});
+        }
+
+    submitUserInformation = () =>{};
 
     render() {
+
+        const signupformStyles = {
```

Comment:
I remember devmountain suggesting this is the easiest way to do styles but I think it is confusing to sprinkle style settings across the site instead of pushing all to the app.css style sheet. Is there a reason we shouldn't have all styles in that single location?